### DISCUSSION TITLE
working with .git being a symlink

### DISCUSSION THREAD

**Discussion by Bhramar-vatsa** [tpope/vim-fugitive#1780] [Q&A] [answered]
In my company there is some area in a remote server that is backed-up, but the space where I work isn't. So, to keep the local work (commits, branches) safe, I have created a symlink for .git to the directory in backed-up area. However, after this, it results into (at least one) fugitive commands not working.
e.g. :Gblame results in an error: "fatal:  <file> is outside repository"
But the 'statusline' does show the git branch.
Note, the usual git works fine with this arrangement. So, is there a way that fugitive can also work in such a case?

**Comment by tpope** [comment, marked as answer]
You need to set `core.worktree`.

**Reply by Bhramar-vatsa** [reply]
Thanks, this works. But git has no issues, it is only with fugitive commands that I am having issues with, actually for now specifically with `Gblame`. There are other commands such as `Git pull` that works fine. So, couldn't there be a way for `Gblame` to work the same way normal git works or even other commands are working?
